log-insights/setup/azure-database/troubleshooting.mdx

```diff
@@ -32,6 +32,7 @@ When using Managed Identity, verify that you have actually assigned the identity
 This may also occur when you have multiple user-assigned managed identities set for your virtual machine.
 Try removing any unrelated managed identities, or explicitly set the `azure_ad_client_id` / `AZURE_AD_CLIENT_ID` setting to
 the Managed Identity's Client ID.
+You may also observe the error message like "ManagedIdentityCredential: no default identity is assigned to this resource" in this case.
```

Comment:
This part is talking about JWT, which we already retired with 0.50.0 but I am lazy to rewrite everything, so I just added a new error message.
The lazy alternative is to update the title from `Error: "failed to configure Azure AD JWT provider"` to `Error: "ManagedIdentityCredential: no default identity is assigned to this resource"` here.
Thoughts?

Reply:
While I think the docs should focus on the latest collector (since this information is most relevant for initial setup, and you're most likely to be setting up a collector right after downloading the latest version), it's helpful to keep troubleshooting information for users of collectors at least a few versions back. They may need to change their configuration and decide to do so without updating the collector and run into problems. I also think it's helpful to "key" troubleshooting information by the error you're seeing.

Given that, is this just a new error message for the same troubleshooting steps? If so, I would change the section title, and add a new section with the old title that just says

> See the above section titled Error: "ManagedIdentityCredential: no default identity is assigned to this resource"

Reply:
Nice, I like that idea. Will update that way 👍